core/src/components/input/test/bottom-content/input.e2e.ts

```diff
@@ -53,7 +53,7 @@ configs({ modes: ['ios'], directions: ['ltr'] }).forEach(({ title, config }) =>
 /**
  * Rendering is the same across modes
  */
-configs({ modes: ['md'], directions: ['ltr'] }).forEach(({ title, screenshot, config }) => {
+configs({ modes: ['md', 'ionic-md'], directions: ['ltr'] }).forEach(({ title, screenshot, config }) => {
```

Comment:
I decided to reuse these tests since at some point, the `ionic` theme will implement the helper text and error text and these tests are testing exactly what needs to be checked.